server/routers/outfit.py

```
@@ -9,19 +13,63 @@ class OutfitRequest(BaseModel):
+        client = get_supabase_client()
+        response = (
+            client.table("profiles")
+            .select("style_preferences")
+            .eq("id", user_id)
+            .single()
+            .execute()
+        )
+        if response.data and response.data.get("style_preferences"):
+            return response.data["style_preferences"].get("default_style", "Casual")
+    except Exception:
+        pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)